raw_description/urdf/vacuum_gripper/gripper.urdf.xacro

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find raw_description)/urdf/vacuum_gripper/gripper.gazebo.xacro" />
+
+  <xacro:property name="gripper_length" value="0.14"/>
+  <xacro:property name="gripper_width" value="0.2"/>
+  <xacro:property name="gripper_height" value="0.14"/>
+  <xacro:property name="gripper_body_height" value="0.14"/>
+  <xacro:property name="radius" value="0.005"/>
+
+  <xacro:macro name="vacuum_gripper" params="name parent *origin">
+
+    <!-- joint between arm_ee_link and raw_boxgripper_link -->
+    <joint name="${name}_base_joint" type="fixed" >
+        <insert_block name="origin" />
+        <parent link="${parent}" />
+        <child link="gripper_frame" />
```

Comment:
you could use the `${name}` parameter here....`<child link="${name}_frame" />`
same for all other occurrences of "gripper"